tools/accuracy_checker/accuracy_checker/postprocessor/brats_postprocessing.py

```diff
@@ -134,3 +134,59 @@ def process_image(self, annotation, prediction):
             target.mask = result
 
         return annotation, prediction
+
+
+class RemoveBratsPredictionPadding(Postprocessor):
+    __provider__ = 'remove_brats_prediction_padding'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'make_argmax': BoolField(
+                optional=True, default=False, description="Allows to apply argmax operation to output values."
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.make_argmax = self.get_value_from_config('make_argmax')
+
+    def process_image(self, annotation, prediction):
+        raise RuntimeError("Since `process_image_with_metadata` is overriden, this method MUST NOT be called")
+
+    def process_image_with_metadata(self, annotation, prediction, image_metadata=None):
+        raw_shape = image_metadata.get('size_after_cropping')
+        if not raw_shape:
+            raise ValueError("No 'size_after_cropping' in dataset metadata")
+
+        for target in prediction:
+
+            # Remove padding
+            padded_shape = target.mask.shape[1:]
+            pad_before = [(p - r) // 2 for p, r in zip(padded_shape, raw_shape)]
+            pad_after = [-(p - r - b) for p, r, b in zip(padded_shape, raw_shape, pad_before)]
+            result = target.mask[:, pad_before[0]:pad_after[0], pad_before[1]:pad_after[1], pad_before[2]:pad_after[2]]
+
+            # Undo cropping
+            bbox = image_metadata.get('crop_bbox')
+            if not bbox:
+                raise ValueError("No 'crop_bbox' in dataset metadata")
+
+            original_size = image_metadata.get('original_size_of_raw_data')
+            if not original_size:
+                raise ValueError("No 'original_size_of_raw_data' in dataset metadata")
```

Comment:
it have relation to specific ndimage, not whole dataset
```suggestion
                raise ValueError("No 'original_size_of_raw_data' in metadata")
```